flink-clients/src/main/java/org/apache/flink/client/program/rest/RestClusterClient.java

```diff
@@ -125,13 +126,18 @@
 import java.util.function.Supplier;
 import java.util.stream.Collectors;
 
+import scala.concurrent.duration.FiniteDuration;
+
 /**
  * A {@link ClusterClient} implementation that communicates via HTTP REST requests.
  */
 public class RestClusterClient<T> extends ClusterClient<T> implements NewClusterClient {
 
 	private final RestClusterClientConfiguration restClusterClientConfiguration;
 
+	/** Timeout for futures. */
+	private final FiniteDuration timeout;
```

Comment:
@TisonKun Maybe it's better to focus on the shifting work in this PR. 
So I'd like keep the type FiniteDuration here and change it to java.time.Duration in the PR of FLINK-14070.

Reply:
Given that FLINK-14070 is on its way, this sounds reasonable. Thanks for your effort!